utils/compress.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
```

Comment:
`#!/usr/bin/env python3`
https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take